metricbeat/module/system/cpu/cpu.go

```diff
@@ -95,6 +95,7 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 			event.Put("softirq.norm.pct", normalizedPct.SoftIRQ)
 			event.Put("steal.norm.pct", normalizedPct.Steal)
 			event.Put("total.norm.pct", normalizedPct.Total)
+			hostFields.Put("host.cpu.pct", normalizedPct.Total)
```

Comment:
All the other metricsets this PR adds `host` data to just constructs a MapStr in `Event()`. I wonder if we want to do that instead for the sake of consistency?